corehq/apps/accounting/static/accounting/ko/accounting.stripe_card_manager.js

```diff
@@ -0,0 +1,153 @@
+var NewStripeCard = function(data){
+    'use strict';
+    var self = this;
+    var mapping = {
+        observe: ['number', 'cvc', 'expMonth','expYear', 'isAutopay', 'token']
+    };
+
+    self.wrap = function(data){
+        ko.mapping.fromJS(data, mapping, self);
+    };
+    self.reset = function(){
+        self.wrap({'number': '', 'cvc': '', 'expMonth': '', 'expYear': '', 'isAutopay': false, 'token': ''});
+    };
+    self.reset();
+
+    self.unwrap = function(){
+        return {token: self.token(), autopay: self.isAutopay()};
```

Comment:
Thanks. I did it this way because I don't need / want to send any of the other cc info to our servers (cc number etc), as this all gets handled by stripe.